portfolio/src/main/java/com/google/sps/data/analysis/SentimentAnalysis.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.data.analysis;
+
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.Sentiment;
+import com.google.sps.common.Constants;
+import java.io.IOException;
+
+public final class SentimentAnalysis {
+
+  public enum TextEvaluation { POSITIVE, NEUTRAL, NEGATIVE; }
```

Comment:
Consider naming this `Sentiment` (or `SentimentCategory` since `Sentiment` is already used by the Cloud Language API) instead of `TextEvaluation`. The latter describes the process of looking at text and coming to some conclusion.